backend/src/main/java/com/woowacourse/friendogly/infra/S3StorageManager.java

```diff
@@ -0,0 +1,68 @@
+package com.woowacourse.friendogly.infra;
+
+import com.woowacourse.friendogly.exception.FriendoglyException;
+import java.io.File;
+import java.io.FileOutputStream;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+import software.amazon.awssdk.auth.credentials.InstanceProfileCredentialsProvider;
+import software.amazon.awssdk.core.sync.RequestBody;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.PutObjectRequest;
+
+@Component
+@Profile("!local")
+public class S3StorageManager implements FileStorageManager {
+
+    @Value("${aws.s3.bucket-name}")
+    private String BUCKET_NAME;
+
+    @Value("${aws.s3.server.endpoint}")
+    private String S3_ENDPOINT;
+
+    @Value("${aws.s3.server.key-prefix}")
+    private String KEY_PREFIX;
+
+    private final S3Client s3Client;
+
+    public S3StorageManager() {
+        this.s3Client = S3Client.builder()
+                .credentialsProvider(
+                        InstanceProfileCredentialsProvider.create())
+                .region(Region.AP_NORTHEAST_2)
+                .build();
+    }
+
+    public String uploadFile(MultipartFile file) {
+        String key = file.getOriginalFilename();
+
```

Comment:
```suggestion
    }
    
    @Override
    public String uploadFile(MultipartFile file) {
        String key = file.getOriginalFilename();

```

없어도 잘 되겠지만, 오버라이드는 명시해주는 게 좋은 것 같아요.